app/assets/stylesheets/application_split2.css

```diff
@@ -0,0 +1,3 @@
+/*
```

Comment:
Never mind! Looks like Mapbox works all the back to IE 8! https://www.mapbox.com/help/mapbox-browser-support/